generator/AWSPSGeneratorLib/HelpMaterials/Examples/DS/Add-DSResourceTag.xml

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--Examples for the AWS PowerShell CmdLets-->
+<examples>
+  <example>
+    <code>$tag = New-Object Amazon.DirectoryService.Model.Tag
+$tag.Key = "myTag"
+$tag.Value = "mytgValue"
+Add-DSResourceTag -ResourceId d-123456ijkl -Tag $tag</code>
```

Comment:
This can be simplified with `Add-DSResourceTag -Tag @{Key="myTag"; Value="mytgValue"}`

Reply:
Simplified as: 
`Add-DSResourceTag -ResourceId d-123456ijkl -Tag @{Key="myTag"; Value="mytgValue"}`